sample/app/controllers/s/CoreOnlySample.scala

```diff
@@ -13,19 +13,19 @@ package controllers.s
 
 import com.linkedin.playparseq.s.PlayParSeq
 import javax.inject.Inject
-import play.api.libs.concurrent.Execution.Implicits._
-import play.api.mvc.{Action, Controller}
-import scala.concurrent.Future
+import play.api.mvc.{Action, AnyContent, Controller}
+import scala.concurrent.{ExecutionContext, Future}
 
 
 /**
  * The class CoreOnlySample is a Controller to show how to use [[PlayParSeq]] for the conversions to ParSeq Task and the
  * execution of ParSeq Task by substring jobs without enabling the ParSeq Trace feature.
  *
  * @param playParSeq The injected [[PlayParSeq]] component
+ * @param executionContext The injected [[ExecutionContext]] component
  * @author Yinan Ding (yding@linkedin.com)
  */
-class CoreOnlySample @Inject()(playParSeq: PlayParSeq) extends Controller {
+class CoreOnlySample @Inject()(playParSeq: PlayParSeq, private[this] implicit val executionContext: ExecutionContext) extends Controller {
```

Comment:
Minor: I think the standard way to do this is have implicits as a separate constructor arg group: https://www.playframework.com/documentation/2.6.x/ThreadPools#using-the-default-thread-pool

Reply:
Good catch— I've only ever seen implicits in their own groups; that's certainly how Play uses them, anyway.

Reply:
Done, updated! Another surprise I learnt.